madcars/runners/localrunner.py

```diff
@@ -7,12 +7,12 @@
 import os
 import sys
 
-maps = ['PillMap']#,'PillHubbleMap', 'PillHillMap', 'PillCarcassMap', 'IslandMap', 'IslandHoleMap']
-cars = ['Buggy']#, 'Bus', 'SquareWheelsBuggy']
```

Comment:
The idea was to narrow a state space to make it easier to train. Can you train the agent well on at least one map?

Reply:
I do. Training should work well on all maps. I have a preference towards learning at all maps/all cars in order to make agent weights more generic.
Otherwise, what I saw is that agent works properly only for a single environment

Reply:
The pushed model does suicide.